pkg/volume/rbd/rbd_util.go

```diff
@@ -55,38 +55,50 @@ const (
 	rbdImageWatcherSteps     = 10
 )
 
-// search /sys/bus for rbd device that matches given pool and image
 func getDevFromImageAndPool(pool, image string) (string, bool) {
+	device, found := getRbdDevFromImageAndPool(pool, image)
+	if found {
+		return device, true
+	}
+	device, found = getNbdDevFromImageAndPool(pool, image)
+	if found {
+		return device, true
+	}
+	return "", false
+}
+
+// Search /sys/bus for rbd device that matches given pool and image.
+func getRbdDevFromImageAndPool(pool string, image string) (string, bool) {
```

Comment:
Just a nit, I think we should use RBD and not Rbd. At least lint will not be happy with Rbd/Nbd